### PR TITLE
Add BQT6 tournament

### PR DESCRIPTION
**Pull request by AcezukyRockon**
Add new BQT tournament and its 6th edition with Schedule information up until Quarter Final and after Semi Final Mappool showcase. 
I use Google Doc to check grammar, I don't have any fluent speaker for review.
This is my first time post wiki, I will try my best to follow the help!

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/BQT/6/en.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+tags:
+  - BQT
+  - BQT6
+---
+
+# BubblyQT 6
+
+![BQT6 logo](img/logo.png)
+
+The **BubblyQT 6** (***BQT6***) is an international team-based double-elimination 2v2 osu! tournament hosted by ::{ flag=US }:: [BubblyQT](https://osu.ppy.sh/users/13414666). This is the sixth installment of the BQT.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:11]:
```suggestion
The **BubblyQT 6** (***BQT6***) is an international team-based double-elimination 2v2 osu! tournament hosted by ::{ flag=US }:: [BubblyQT](https://osu.ppy.sh/users/13414666). It is the sixth installment of the BQT tournament series.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+| **little batos** | ::{ flag=BR }:: **[Nashik](https://osu.ppy.sh/users/12996332)**, ::{ flag=US }:: [ITotallyGetThat](https://osu.ppy.sh/users/8770622), ::{ flag=CL }:: [Neicho](https://osu.ppy.sh/users/18075313), ::{ flag=CL }:: [-Naroh](https://osu.ppy.sh/users/9087831) |
+| **ronos inc.** | ::{ flag=BR }:: **[Doczera-](https://osu.ppy.sh/users/8155626)**, ::{ flag=BR }:: [minimundos](https://osu.ppy.sh/users/16768757), ::{ flag=US }:: [ShorkBork](https://osu.ppy.sh/users/9364519), ::{ flag=BR }:: [AriMaki](https://osu.ppy.sh/users/2129975), ::{ flag=BR }:: [Mestre](https://osu.ppy.sh/users/11227346) |
+| **Pls Enjoy Game** | ::{ flag=GB }:: **[Velyro](https://osu.ppy.sh/users/16765307)**, ::{ flag=US }:: [affirmedcheese](https://osu.ppy.sh/users/21002718), ::{ flag=NZ }:: [Decayed](https://osu.ppy.sh/users/18821604), ::{ flag=KR }:: [Hoper22](https://osu.ppy.sh/users/16193374), ::{ flag=NZ }:: [Pegwin](https://osu.ppy.sh/users/19990127) |
+
+## Mappools
+
+### Semi Finals
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:106]:
```suggestion
### Semifinals
```

File: wiki/Tournaments/BQT/6/en.md [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+  3. [lical - nyctalopia (FlatPaper) \[hard\]](https://osu.ppy.sh/beatmapsets/1971703#osu/4091132)
+  4. [Ceres Fauna - Let Me Stay Here (Gweon Sua) \[A Fairy Tale For Saplings\]](https://osu.ppy.sh/beatmapsets/1766945#osu/3616922)
+- FreeMod
+  1. [Slax - Sugar Loli (smolship) \[Expert\]](https://osu.ppy.sh/beatmapsets/1894197#osu/3903479)
+  2. [Electric Six - Gay Bar (Lutschkas) \[just anime peeps in a bar\]](https://osu.ppy.sh/beatmapsets/2078871#osu/4353076)
+  3. [Ryokuoushoku Shakai - Mela! (Battle) \[Hero!\]](https://osu.ppy.sh/beatmapsets/1209920#osu/2519013)
+- Tiebreaker
+  1. **[FantasticPlanets - Jekyll & Hyde (Wishkey) \[Abyss\]](https://osu.ppy.sh/beatmapsets/910518#osu/1899986)**
+
+### Quarter Finals
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:137]:
```suggestion
### Quarterfinals
```

File: wiki/Tournaments/BQT/6/en.md [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+  1. [S3RL - Bass Slut (Original Mix) (Secretpipe) \[Kyshiro's Extra\]](https://osu.ppy.sh/beatmapsets/375648#osu/836907)
+  2. [TAROLIN feat. Katakiri Rekka - Cardinal Rave (Yusomi) \[Cub's Insane\]](https://osu.ppy.sh/beatmapsets/1116399#osu/2355610)
+- DoubleTime
+  1. [Hanasaka Yui(CV: M.A.O) - Harumachi Clover (ezek) \[Fiery's Extra\]](https://osu.ppy.sh/beatmapsets/652412#osu/1396152)
+  2. [Kijibato - w/WWW feat. Hoshimiya Toto (Elcheer) \[Hard\]](https://osu.ppy.sh/beatmapsets/1312343#osu/2855311)
+  
+
+## Match results
+
+### Quarter Finals
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:242]:
```suggestion
### Quarterfinals
```

File: wiki/Tournaments/BQT/6/en.md [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+| **:holaaaaaaa:** | **5** | 3 | ronos inc. | [#1](https://osu.ppy.sh/community/matches/110803945) |
+| **CookedCookiesSquared** | **5** | 3 | -7 +9 | [#1](https://osu.ppy.sh/community/matches/110803949) |
+
+## Ruleset
+
+### Tournament rules
+
+1. This is an International 2v2 • osu!standard tournament
+2. Minimum number of players per team is 2, with a maximum of 4
+3. Matches will be Team Vs and use ScoreV2
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:332]:
```suggestion
3. All matches will be done under the `Team Vs` setting and the [ScoreV2](https://osu.ppy.sh/wiki/en/Gameplay/Score#scorev2) scoring system.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 6]
```diff
@@ -0,0 +1,389 @@
...
+| **floppy** | **5** | 1 | Mika's Cake Roll | [#1](https://osu.ppy.sh/community/matches/110801914) |
+| **:holaaaaaaa:** | **5** | 3 | ronos inc. | [#1](https://osu.ppy.sh/community/matches/110803945) |
+| **CookedCookiesSquared** | **5** | 3 | -7 +9 | [#1](https://osu.ppy.sh/community/matches/110803949) |
+
+## Ruleset
+
+### Tournament rules
+
+1. This is an International 2v2 • osu!standard tournament
+2. Minimum number of players per team is 2, with a maximum of 4
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:331]:
```suggestion
2. Teams are required to have 2 to 4 team members each in order to participate.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 7]
```diff
@@ -0,0 +1,389 @@
...
+3. Matches will be Team Vs and use ScoreV2
+4. Rank range for the tournament: Open Rank (Must been a player in BQT1, BQT2, BQT3, BQT4, BQT5 to be eligible)
+5. Players are required to be in the discord for the duration of the tournament until their team is completely eliminated.
+6. BQT1-5 players are eligible to play in BQT6.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:335]: delete this rule; already covered by Rule 4

File: wiki/Tournaments/BQT/6/en.md [hunk 8]
```diff
@@ -0,0 +1,389 @@
...
+| **CookedCookiesSquared** | **5** | 3 | -7 +9 | [#1](https://osu.ppy.sh/community/matches/110803949) |
+
+## Ruleset
+
+### Tournament rules
+
+1. This is an International 2v2 • osu!standard tournament
+2. Minimum number of players per team is 2, with a maximum of 4
+3. Matches will be Team Vs and use ScoreV2
+4. Rank range for the tournament: Open Rank (Must been a player in BQT1, BQT2, BQT3, BQT4, BQT5 to be eligible)
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:333]:
```suggestion
4. Although the tournament doesn't exercise a strict rank restriction, players must have been participated before in BQT1, BQT2, BQT3, BQT4, or BQT5 in order to be eligible.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 9]
```diff
@@ -0,0 +1,389 @@
...
+5. Players are required to be in the discord for the duration of the tournament until their team is completely eliminated.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:334]:
```suggestion
5. Players are required to be in the Discord for the duration of the tournament as long as their team is participating in the tournament.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 10]
```diff
@@ -0,0 +1,389 @@
...
+6. BQT1-5 players are eligible to play in BQT6.
+7. This tournament will have Qualifiers ➔ RO32 ➔ RO16 ➔ Quarterfinals ➔ Semifinals ➔ Finals ➔ Grand Finals
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:336]:
```suggestion
7. The tournament will be conducted in seven stages, namely Qualifiers ➔ RO32 ➔ RO16 ➔ Quarterfinals ➔ Semifinals ➔ Finals ➔ Grand Finals.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 11]
```diff
@@ -0,0 +1,389 @@
...
+8. There will be a bracket reset in the Grand Finals if necessary
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:337]:
```suggestion
8. There will be a bracket reset in the Grand Finals if necessary.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 12]
```diff
@@ -0,0 +1,389 @@
...
+9. Semifinals, Finals, and Grand Finals will be streamed. Other matches will TBD depending on the streamers' availability.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:338]:
```suggestion
9. All matches from the Semifinals onwards will be streamed. Other matches are also subject to be streamed depending on the streamers' availability.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 13]
```diff
@@ -0,0 +1,389 @@
...
+10. All match times are in UTC. Find your own time zone here if you're unsure what UTC you're in.
+11. Only BQT Refs, Streamers, and Casters are allowed to play in the tournament, but they are referees, they can't ref their own match.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:340]:
```suggestion
11. Referees, streamers, and match casters are allowed to play in the tournament. However, referees may not self-referee their own match.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 14]
```diff
@@ -0,0 +1,389 @@
...
+12. 6 Digit badged players are ALLOWED to participate.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:341]: this rule should also be deleted as it's already covered by Rule 4

File: wiki/Tournaments/BQT/6/en.md [hunk 15]
```diff
@@ -0,0 +1,389 @@
...
+13. Team Supporter!
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:342]:
```suggestion
13. The tournament allows for each team to assign dedicated Team Supporters for their team, which comes with the following benefits:
```

File: wiki/Tournaments/BQT/6/en.md [hunk 16]
```diff
@@ -0,0 +1,389 @@
...
+     
+   - **Other Helpful Information:**
+
+     - Any rank is allowed to be a Team Supporter!
+     - No incentive! Just wanting to give back to the osu! Community! You're like a coach or guidance for their tournament experience!
+     - If we know you're being offensive, aggressive, and rude to your and others teams, you'll get 2 warnings and the last one would be a ban from the server. Positive Vibes Only!
+
+### Qualifier instructions
+
+1. Qualifiers will be played to determine who QUALIFIES (TOP 32) as well as TEAM SEEDING
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:363]:
```suggestion
1. Qualifiers will be played to determine who QUALIFIES (TOP 32) as well as TEAM SEEDING.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 17]
```diff
@@ -0,0 +1,389 @@
...
+2. Every team must have **AT LEAST 2 members** present to their qualifier session to be eligible. If you have a 3rd/4th player in your team, they may sub in for the map they're playing for.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:364]:
```suggestion
2. Each team must have **AT LEAST 2 members** present to their qualifier session to participate. Extra players may be substituted in and out freely from the lobby in between the maps.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 18]
```diff
@@ -0,0 +1,389 @@
...
+3. Set Times for Qualifier Lobbies will be shown on Mainsheet. Each team may only REGISTER for ONE LOBBY.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:365]:
```suggestion
3. Set times for Qualifier Lobbies will be shown on the Main Sheet. Each team may only REGISTER for ONE LOBBY.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 19]
```diff
@@ -0,0 +1,389 @@
...
+4. If you miss your scheduled lobby, you may choose a different lobby as long as there is still space.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:366]:
```suggestion
4. If a team misses their scheduled lobby, the captain of the team may register the team for a different lobby as long as there is still another lobby available.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 20]
```diff
@@ -0,0 +1,389 @@
...
+5. If your team CANNOT meet at any of the scheduled times, please DM/Ping @Admin to arrange a custom lobby, however this cannot be guaranteed based on the schedules of staff.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:367]:
```suggestion
5. If a team CANNOT meet at any of the scheduled times, please DM/Ping @Admin to arrange a custom lobby. Please note that the custom lobbies are not guaranteed as they are based on the staff availability.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 21]
```diff
@@ -0,0 +1,389 @@
...
+6. If a player disconnects during the first 30 seconds of the map, that specific player must stay at the end of the lobby and replay that map by themselves.
+
+### Match instructions
+
+1. Matches are made approximately 5-10 MINUTES before the matches are scheduled to start.
+2. (Team 1) vs (Team 2) = Team 1 must be in Slot 1-2 (Red Team Color) | Team 2 must be in Slot 3-4 (Blue Team Color)
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:373]:
```suggestion
2. All match lobbies will be named `(Team 1) vs (Team 2)`, with players from Team 1 playing in slots #1 and #2 (Red Team) and players from Team 2 playing in slots #3 and #4 (Blue Team).
```

File: wiki/Tournaments/BQT/6/en.md [hunk 22]
```diff
@@ -0,0 +1,389 @@
...
+7. Each team will be allowed 60 seconds to ban their map. When 2 bans are enabled, teams are **NOT allowed to ban from the same mod** (NM is the exception). For example, Team 1 can't ban both HD1 and HD2 during their banning phase.
+8. Picks will happen in alternating order after the previous map is finished. Each team has 120 seconds to pick their map. If a map is not picked by 120 seconds, the pick will go to the other team.
+9. **Double picking is NOT ALLOWED**, except for NoMods. For example, Team 1 can't pick HR1 then pick HR2 on their next turn. They can pick HR1, DT1, then they can pick HR2.
+10. A map will be aborted if a player disconnects within the first 30 seconds of the map. If there is a disconnect and the map is not aborted, the score for that player will be counted as 0 UNLESS a score is provided. AN ABORT IS ALLOWED ONCE PER TEAM, PER MATCH.
+11. NoFail will be enforced on every map.
+12. Freemod rules : Each players in a team must have a unique mod! One player with HD/EZ/HDEZ and another player with HR/HDHR . EZ = 1.8x (with potentially HD)
+13. Players are not required to take any mods on the tiebreaker. However, they are allowed to without penalty.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:384]:
```suggestion
13. Players are not required to take any mods on the tiebreaker map. However, playing the tiebreaker map with mods is not prohibited.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 23]
```diff
@@ -0,0 +1,389 @@
...
+3. If there are less than 2 players from either team WITHIN THE LOBBY, there is a 15 minute grace period provided to allow them to join; **10 minutes - team lose ban**, **15 minute - FF**. The team with the most players within the lobby by the 15 minute mark will win by default. If both teams have only one player present on each side and both of their 2nd teammate doesn't show up by the 15 minute grace period, it'll be FF on both sides.
+4. There will be **NO WARMUPS** this season, so please warmup before your match.
+5. Each team will use `!roll` for the banning and picking phase. The winner of the roll will get to **choose pick order/ban order** The loser of the roll will get to choose between the unchosen options. For Example... If Team 1 rolls 100 and Team 2 rolls 50... Team 1 wants to choose ban order & they may choose ban first or second...Then that leaves Team 2 to choose from pick order. They may choose to pick first or second.
+6. 1 BAN ALLOWED PER TEAM. This rule is applied up to RO16. Afterwards, 2 BANS ALLOWED PER TEAM (ABBA). **Double Banning Mods are not allowed**, except for NoMods.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:377]:
```suggestion
6. There will be 1 BAN PER TEAM on matches up to RO16, and 2 BANS PER TEAM (ABBA format) on matches from Quarterfinals onwards. **Each team may not ban two maps from the same mod pool in a match**, except for NoMod maps.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 24]
```diff
@@ -0,0 +1,389 @@
...
+7. Each team will be allowed 60 seconds to ban their map. When 2 bans are enabled, teams are **NOT allowed to ban from the same mod** (NM is the exception). For example, Team 1 can't ban both HD1 and HD2 during their banning phase.
+8. Picks will happen in alternating order after the previous map is finished. Each team has 120 seconds to pick their map. If a map is not picked by 120 seconds, the pick will go to the other team.
+9. **Double picking is NOT ALLOWED**, except for NoMods. For example, Team 1 can't pick HR1 then pick HR2 on their next turn. They can pick HR1, DT1, then they can pick HR2.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:380]:
```suggestion
9. **Picking maps from the same mod pool consecutively in a match is NOT ALLOWED**, except for NoMod maps. For example, when a team picks HR1, they can't pick HR2 in their next turn. They had to pick something else first (e.g. DT1) before they can pick HR2.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 25]
```diff
@@ -0,0 +1,389 @@
...
+10. A map will be aborted if a player disconnects within the first 30 seconds of the map. If there is a disconnect and the map is not aborted, the score for that player will be counted as 0 UNLESS a score is provided. AN ABORT IS ALLOWED ONCE PER TEAM, PER MATCH.
+11. NoFail will be enforced on every map.
+12. Freemod rules : Each players in a team must have a unique mod! One player with HD/EZ/HDEZ and another player with HR/HDHR . EZ = 1.8x (with potentially HD)
+13. Players are not required to take any mods on the tiebreaker. However, they are allowed to without penalty.
+
+### Scheduling instructions
+
+1. Every week, the administrators will create reschedule channels in Discord for each matches. Both teams of each match will cooperate to find a time and date for their matches then post on their channel.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:388]:
```suggestion
1. Each week, the Tournament Management will create a scheduling-specific channels in the Discord server for every match. Both teams may cooperate to find an alternative time and date for their match in the corresponding channel.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 26]
```diff
@@ -0,0 +1,389 @@
...
+2. For loser bracket potential rounds, the deadline to reschedule their first match is by Thursday each week, and the deadline to reschedule their second match is by Friday each week.
+3. Administrators has the right to decline reschedule requests as well help the teams to reschedule their matches.
```
Comment (Nivalyx) [inline wiki/Tournaments/BQT/6/en.md:390]:
```suggestion
3. The Tournament Management has the right to decline reschedule requests as well as to offer an optimal match time for the teams.
```

**Comment by AcezukyRockon** [comment]
Hello, I just want to say thank you for a lot of suggestions. 

I have realized that the suggestions are made so that players or spectators who have no tournament experiences can still understand.

It really helps me to write or suggests to the Host a better Tournament Rules for some future tournaments (especially for the rules are written in Forum Post). 
I hope I can remember these so I don't have to bug you every time I make a Wiki for a tournament (if there is one).

File: wiki/Tournaments/BQT/6/en.md [hunk 27]
```diff
@@ -0,0 +1,388 @@
...
+6. There will be 1 BAN PER TEAM on matches up to RO16, and 2 BANS PER TEAM (ABBA format) on matches from Quarterfinals onwards. **Each team may not ban two maps from the same mod pool in a match**, except for NoMod maps.
+7. Each team will be allowed 60 seconds to ban their map. When 2 bans are enabled, teams are **NOT allowed to ban from the same mod** (NM is the exception). For example, Team 1 can't ban both HD1 and HD2 during their banning phase.
+8. Picks will happen in alternating order after the previous map is finished. Each team has 120 seconds to pick their map. If a map is not picked by 120 seconds, the pick will go to the other team.
+9. **Picking maps from the same mod pool consecutively in a match is NOT ALLOWED**, except for NoMod maps. For example, when a team picks HR1, they can't pick HR2 in their next turn. They had to pick something else first (e.g. DT1) before they can pick HR2.
+10. A map will be aborted if a player disconnects within the first 30 seconds of the map. If there is a disconnect and the map is not aborted, the score for that player will be counted as 0 UNLESS a score is provided. AN ABORT IS ALLOWED ONCE PER TEAM, PER MATCH.
```
Comment (Hiviexd) [inline wiki/Tournaments/BQT/6/en.md:379]: i don't really get the `unless a score is provided` part? does this mean the player has to quickly make a singleplayer score on the map when the match is still ongoing, and then provide the replay to the staff?

File: wiki/Tournaments/BQT/6/en.md [hunk 28]
```diff
@@ -0,0 +1,388 @@
...
+11. NoFail will be enforced on every map.
+12. Freemod rules : Each players in a team must have a unique mod! One player with HD/EZ/HDEZ and another player with HR/HDHR . EZ = 1.8x (with potentially HD)
```
Comment (Hiviexd) [inline wiki/Tournaments/BQT/6/en.md:381]:
```suggestion
12. When playing FreeMod, each player in a team must have a unique mod! One player with HD/EZ/HDEZ and another player with HR/HDHR. EZ = 1.8x (with potentially HD)
```

wording still feels lackluster so i'd appreciate another maintainer's take on how to adjust this.

File: wiki/Tournaments/BQT/6/en.md [hunk 29]
```diff
@@ -0,0 +1,388 @@
...
+13. Players are not required to take any mods on the tiebreaker map. However, playing the tiebreaker map with mods is not prohibited.
```
Comment (Hiviexd) [inline wiki/Tournaments/BQT/6/en.md:382]:
```suggestion
13. Players are not required to take any mods on the tiebreaker map. However, playing the tiebreaker map with mods is allowed.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 30]
```diff
@@ -0,0 +1,388 @@
...
+
+### Scheduling instructions
+
+1. Each week, the Tournament Management will create a scheduling-specific channels in the Discord server for every match. Both teams may cooperate to find an alternative time and date for their match in the corresponding channel.
```
Comment (Hiviexd) [inline wiki/Tournaments/BQT/6/en.md:386]:
```suggestion
1. Each week, the organisation team will create scheduling-specific channels in the Discord server for every match. Both teams may cooperate to find an alternative time and date for their match in the corresponding channel.
```

File: wiki/Tournaments/BQT/6/en.md [hunk 31]
```diff
@@ -0,0 +1,388 @@
...
+2. For loser bracket potential rounds, the deadline to reschedule their first match is by Thursday each week, and the deadline to reschedule their second match is by Friday each week.
+3. The Tournament Management has the right to decline reschedule requests as well as to offer an optimal match time for the teams.
```
Comment (Hiviexd) [inline wiki/Tournaments/BQT/6/en.md:388]:
```suggestion
3. The organisation team has the right to decline reschedule requests as well as to offer an optimal match time for the teams.
```

File: wiki/Tournaments/BQT/6/en.md (hunk 27 above)
Comment (AcezukyRockon) [reply wiki/Tournaments/BQT/6/en.md:379]: Sometime a player is disconnected during playing a map, but they can still continue play "in offline" and completed their play and have result screen, they can screenshot this result screen and show to referee. 

Or they have problem connect to Bancho but their internet is still fine,  and if they live on the twitch play, they can send twitch clip for example.

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**